packages/components/molecules/f-mega-modal/README.md

```diff
@@ -139,59 +92,20 @@ In addition to this, the modal exposes some classes which you can target in your
 | `open` | This event is emitted when the modal is opened. |
 | `close` | This event is emitted when the modal is closed. |
 
-You can add event listeners for these like so
```

Comment:
What's the reason for deleting this block?

Reply:
I think if we want to remove it, we should replace it with a doc in storybook replacing it before we do so, so we can reference it from here.

Can you make a ticket to do this and put this back in until then?